helpers.py

```diff
@@ -37,6 +37,12 @@ def put(conn, file, key_filename=None, local_path=".", remote_path="."):
     conn.local("scp %s -C -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -P %s %s %s@%s:%s" %
           (key_arg, conn.port, os.path.join(local_path, file), conn.user, conn.host, remote_path))
 
+def run(conn, command, key_filename=None, warn=False):
+    key_arg = _prepare_key_arg(key_filename)
+    result = conn.local("ssh %s -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=60 -p %s %s@%s %s" %
```

Comment:
I think 60 should be raised to maybe 120 or 180. The boot is already easily 20-30 seconds, and with a bit of load you could exceed 60 seconds.

Reply:
Rising it to 120 then. The boot has a different place on where to probe the SSH connection

Reply:
Well, wait. I originally selected 60 because it is also the timeout we are setting for the Fabric SSH connections, see [helpers.py](https://github.com/mendersoftware/mender-test-containers/blob/master/helpers.py#L94).

Unless you strongly disagree, I will leave it as it is for consistency between both methods.

Reply:
Oh ok, sure. I thought it was much higher, but maybe that is only in integration.